community-website/src/community-project-boilerplate-docgen/src/widgets/search-box.md

```diff
@@ -11,13 +11,13 @@ The Search Box component displays a search box that lets the user search for a s
 
 ## Options
 
-| Attribues         | Type      | Description
-| -                 | -         | -
-| `placeholder?`    | `string`  | The label of the input placeholder.
-| `submitTitle`     | `string`  | The submit button title.
-| `resetTitle`      | `string`  | The reset button title.
-| `searchAsYouType` | `boolean` | If you disable this option, new searches will only be triggered by clicking the search button or by pressing the enter key while the search box is focused.
-| `autofocus?       | `boolean` | Should the search box be focused on render?.
+| Attribues          | Type      | Description
+| -                  | -         | -
+| `placeholder?`     | `string`  | The label of the input placeholder.
+| `submitTitle?`     | `string`  | The submit button title.
+| `resetTitle?`      | `string`  | The reset button title.
+| `searchAsYouType?` | `boolean` | If you disable this option, new searches will only be triggered by clicking the search button or by pressing the enter key while the search box is focused.
```

Comment:
should be mentioned that the default is `true` (can be left for the docs on algolia.com, doesn't need to be done now)